features/bevragen/persoon/adressering/aanhef/dev/gba.feature

```diff
@@ -2,4 +2,6 @@ Functionaliteit: specifieke scenario's voor de gba-API
 
 Rule: Als de fields een Aanhef bevat dan moet de GBA-API een set LO-GBA velden leveren. 
 
-    Scenario: Lever de volgende velden bij vragen om aanhef
\ No newline at end of file
+  Scenario: Lever de volgende velden bij vragen om adressering.aanhef : naam.geslacht, partners.naam.adellijkeTitelPredicaat, partners.naam.voorvoegsel, partners.naam.geslachtsnaam, partners.aangaanHuwelijkPartnerschap.datum, partners.ontbindingHuwelijkPartnerschap.datum
+
+# Vraag: dit gaat alleen om het leveren van de velden zoals die in field-gba-mapping zijn gedefinieerd. Zetten we dat hier neer ? Of komt daar een generieke fields feature voor ? 
```

Comment:
ik zou de field mapping scenario's hier toevoegen. Reden: makkelijker kunnen terug vinden welke mapping moet worden toegepast

Reply:
Ik zou een scenario dat specifiek hieraan is gewoon plaatsen. Dat wil niet zeggen dat er geen CSV of zo iets dergelijks hoeft te komen.

Reply:
Toegevoegd